README.md

```diff
@@ -2,6 +2,9 @@
 
 Fable bindings for virtual-dom
 
+## Website
+[fable.io/fable-arch](http://fable.io/fable-arch/#/)
```

Comment:
After more thinking I think that it may be even better that the first title should also be a link to the website. That way, we cannot miss it (believe it or not but I missed the one in the description at the top)* and it would take less place in the Readme.